doc-surrealdb_versioned_docs/version-latest/querying/surrealql/http.mdx

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 2
+sidebar_label: SurrealQL via HTTP 
+title: SurrealQL via HTTP | SurrealQL
+description: In this section, you will explore querying SurrealDB using HTTP. The HTTP API is designed to be simple and intuitive, with a RESTful interface that provides a consistent way to interact with the database.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# SurrealQL via HTTP
+
+SurrealDB provides a powerful HTTP API that allows you to interact with the database programmatically. This API can be used to perform a wide range of database operations, from querying data to modifying records and managing database structures.
+
+The HTTP API is designed to be simple and intuitive, with a RESTful interface that provides a consistent way to interact with the database. You can use the API to perform a wide range of database operations, from querying data to modifying records and managing database structures.
+
+
+
+
+## Using curl `POST /sql`
+
+The SQL endpoint enables use of SurrealQL queries.
```

Comment:
The /sql endpoint, I think (avoid misinterpretation that this endpoint is for traditional SQL)